ariadne/cardano/src/Ariadne/Wallet/Cardano/WalletLayer/Types.hs

```diff
@@ -90,4 +90,7 @@ data PassiveWalletLayer m = PassiveWalletLayer
     , pwlLookupKeystore
           :: Kernel.HdRootId
           -> m (Maybe EncryptedSecretKey)
+    , pwlRemoveKey
```

Comment:
I don't like this name, how about `pwlRemoveUnknownKeys`? What I don't like:
1. `Key` is singular, though it deletes multiple keys.
2. This function shouldn't be used to delete an arbitrary key, because deletion is normally done by `pwlDeleteWallet`. It's only for a very specific case, so its name should reflect that.
Also you put it into the `internal, hopefully these will go in the future` group, but it's not really internal and we need it to go in the future.

Reply:
So?